Build error in msys2 (mingw64 mode)

The same error is "  __attribute_used__ = { 0x7ff8000000000000 };"
**I used gcc 7.3.0 cmake 3.12.0.**
E:/msys64/home/heche/io/libs/iovm/source/IoNumber.h:27:2: error: expected '=',  ,', ';', 'asm' or '__attribute__' before '__attribute_used__'
  __attribute_used__ = { 0x7ff8000000000000 };
  ^~~~~~~~~~~~~~~~~~
In file included from E:/msys64/home/heche/io/libs/iovm/source/IoBlock.c:14:0:
E:/msys64/home/heche/io/libs/iovm/source/IoNumber.h:27:2: error: expected '=',  ,', ';', 'asm' or '__attribute__' before '__attribute_used__'
  __attribute_used__ = { 0x7ff8000000000000 };
  ^~~~~~~~~~~~~~~~~~
In file included from E:/msys64/home/heche/io/libs/iovm/source/IoCoroutine.c:14:0:
E:/msys64/home/heche/io/libs/iovm/source/IoNumber.h:27:2: error: expected '=',  ,', ';', 'asm' or '__attribute__' before '__attribute_used__'
  __attribute_used__ = { 0x7ff8000000000000 };
  ^~~~~~~~~~~~~~~~~~
In file included from E:/msys64/home/heche/io/libs/iovm/source/IoCollector.c:27:0:
E:/msys64/home/heche/io/libs/iovm/source/IoNumber.h:27:2: error: expected '=',  ,', ';', 'asm' or '__attribute__' before '__attribute_used__'
  __attribute_used__ = { 0x7ff8000000000000 };
  ^~~~~~~~~~~~~~~~~~
In file included from E:/msys64/home/heche/io/libs/iovm/source/IoCompiler.c:10: :
E:/msys64/home/heche/io/libs/iovm/source/IoNumber.h:27:2: error: expected '=',  ,', ';', 'asm' or '__attribute__' before '__attribute_used__'
  __attribute_used__ = { 0x7ff8000000000000 };
  ^~~~~~~~~~~~~~~~~~
In file included from E:/msys64/home/heche/io/libs/iovm/source/IoDate.c:13:0:
E:/msys64/home/heche/io/libs/iovm/source/IoNumber.h:27:2: error: expected '=',  ,', ';', 'asm' or '__attribute__' before '__attribute_used__'
  __attribute_used__ = { 0x7ff8000000000000 };
  ^~~~~~~~~~~~~~~~~~

--- a/libs/iovm/source/IoNumber.h
+++ b/libs/iovm/source/IoNumber.h
@@ -23,8 +23,7 @@ extern "C" {
  * So 0/0, -0/0, 0/-0, -0/-0 all give -NaN.
  */
 #if defined(USE_BUILTIN_NAN) || !defined(NAN)
-static union { unsigned long long __c; double __d; } __nan_union
-	__attribute_used__ = { 0x7ff8000000000000 };
+static union { unsigned long long __c; double __d; } __nan_union = { 0x7ff8000000000000 };
 #undef NAN
 #define NAN (__nan_union.__d)
 #endif

